mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -427,7 +443,7 @@ def WaitForGetLatestDetectionResultImages(self, timeout=2.0):
         return self._WaitForResponse(recvjson=False, timeout=timeout)
     
     def GetDetectionHistory(self, timestamp, timeout=2.0):
-        # type: (float, float) -> typing.Any
+        # type: (float, float) -> Any
```

Comment:
Docstring suggest that timestamp should be `timestamp (int): unix timestamp in milliseconds`

Reply:
That's correct afaik, the type hint was wrong